ruby/overrides.js

```diff
@@ -45,4 +45,5 @@
     'should waitBeforeCapture with breakpoints in check': { skipEmit: true },
     'should waitBeforeCapture with breakpoints in open': { skipEmit: true },
     'should be empty if page delayed by 1500': { skipEmit: true },
+    'appium iOS nav bar check regio': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9214167)
```suggestion
    "appium iOS nav bar check regio": {skipEmit: true},
```